cmd/crictl/image.go

```diff
@@ -121,15 +125,25 @@ var listImageCommand = cli.Command{
 				continue
 			}
 			if !verbose {
+				showDigest := context.Bool("digests")
 				if printHeader {
 					printHeader = false
-					fmt.Fprintln(w, "IMAGE\tTAG\tIMAGE ID\tSIZE")
+					if showDigest {
+						fmt.Fprintln(w, "IMAGE\tTAG\tDIGEST\tIMAGE ID\tSIZE")
+					} else {
+						fmt.Fprintln(w, "IMAGE\tTAG\tIMAGE ID\tSIZE")
+					}
 				}
+				repoDigest := normalizeRepoDigest(image.RepoDigests)
 				repoTagPairs := normalizeRepoTagPair(image.RepoTags, image.RepoDigests)
 				size := units.HumanSizeWithPrecision(float64(image.GetSize_()), 3)
 				trunctedImage := strings.TrimPrefix(image.Id, "sha256:")[:truncatedImageIDLen]
 				for _, repoTagPair := range repoTagPairs {
```

Comment:
We should combine repoDigest list and repoTag list, so that we don't need to handle repo digest separately in `normalizeRepoTagPair`.

Reply:
done